general/src/protocol/span.rs

```diff
@@ -0,0 +1,64 @@
+use chrono::{DateTime, Utc};
+
+use crate::protocol::{SpanId, TraceId};
+use crate::types::{Annotated, Object, Value};
+
+// TODO typing ( union of predefined types and custom 'string' types)
+type OperationType = String;
```

Comment:
_nit_: Please add a doc comment.

Reply:
rls said we can't have a doccomment here.

Reply:
wat. Also I'm wondering if this should be public. Since it's an alias to a prelude type this will not error, but this is weird behavior.

Reply:
I'm guessing as soon as you make it `pub` you can put a doc comment there.

Reply:
I think we should make it public once it's an actual newtype

Reply:
meh, done